source/ota.c

```diff
@@ -2231,22 +2214,28 @@ static void handleJobParsingError( const OtaFileContext_t * pFileContext,
                         "OtaJobParseErr_t=%s, Job name=%s",
                         OTA_JobParse_strerror( err ), ( const char * ) pFileContext->pJobName ) );
 
-            /* Assume control of the job name from the context. */
-            ( void ) memcpy( otaAgent.pActiveJobName, pFileContext->pJobName, OTA_JOB_ID_MAX_SIZE );
+            if( strlen( ( const char * ) otaAgent.pActiveJobName ) > 0u )
```

Comment:
A new test was added here, but I don't see any unit test code added for this.  Can we add unit test coverage for this?